katsdpimageutils/test/test_primary_beam_correction.py

```diff
@@ -0,0 +1,111 @@
+"""Tests for :mod:`katsdpimageutils.primary_beam_correction`."""
+import shutil
+import os
+
+import numpy as np
+from astropy.io import fits
+from nose.tools import assert_equal
+
+from ..primary_beam_correction import _get_value_from_history, weighted_average, inverse_variance, standard_deviation, write_new_fits # noqa E501 
+
+
+class TestStandardDeviation():
+    def setup(self):
+        self.arr = np.array([2, 4, 6, 8, 10]).astype(float)
+
+    def test_nonans(self):
+        data = self.arr
+        med, sd = standard_deviation(data)
+        assert_equal([med, sd], [6.0, 2.9652])
```

Comment:
This is just a suggestion but you could write the standard deviation answer as a multiple of the MAD_TO_SD constant, that way its more obvious where the calculated answer comes from and its easier for a new person reading the code to check the answer. eg. `assert_equal([med, sd]), [6.0, 2 * 1.4826])`
You could do this for the other places that use this constant

Reply:
Noted and implemented.